scripts/ynh_package_version

```diff
@@ -15,6 +15,7 @@ abort_if_up_to_date () {
     version=$(read_json "/etc/yunohost/apps/$YNH_APP_INSTANCE_NAME/manifest.json" 'version' 2> /dev/null || echo '20160501-7')
     last_version=$(read_manifest 'version')
     if [ "${version}" = "${last_version}" ]; then
+        sleep 60
```

Comment:
uuuuuh, ... 60, really o.O ? To ... do nothing O.ô ?

![serveimage](https://user-images.githubusercontent.com/4533074/67553525-aff1b680-f70d-11e9-9d4d-c4ac0526dc71.gif)

Reply:
I know.. I'm trying to figure why I have an error "502 Bad gateway" at upgrade check on my own CI...
All other packages go well, even some package not working on DEV CI, work on my CI....

Reply:
And now there is the same error on CI-APPS: https://ci-apps.yunohost.org/ci/job/5412 :'/